src/styles/blocks/cards.scss

```
@@ -0,0 +1,137 @@
+.cards {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  gap: 44px 46px;
+  margin: 50px 40px;
+
+  @media (max-width: 1023.98px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (max-width: 767.98px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (max-width: 487.98px) {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+.product {
+  position: relative;
+  max-width: 200px;
+  cursor: pointer;
+  transition: color $transition, transform $transition;
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  padding: 32px 16px 16px;
+  border: 1px solid $borfer-color;
+  border-radius: 5px;
+
+  &__img {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__img-img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    margin: 0 0 4px;
+    font-weight: 500;
+    color: $main-color;
+  }
+
+  &__name,
+  &__price-title {
+    font-size: 12px;
+    line-height: 150%;
```

use `px`  in all cases
```suggestion
    line-height: ...;
```